src/midi/midi_fluidsynth.h

```diff
@@ -40,7 +40,7 @@ class MidiHandlerFluidsynth final : public MidiHandler {
 	~MidiHandlerFluidsynth() override;
 	void PrintStats();
 
-	std::string_view GetName() const override
+	std::string GetName() const override
```

Comment:
Good riddance! Returning `std::string_view` from classes/modules always makes me feel uneasy...

Reply:
I didn't explicitly look for these, but they are a "nice" candidate for a future PR.

Reply:
Even better: looks like this was the only instance where std::string_view was returned.

Reply:
> Even better: looks like this was the only instance where std::string_view was returned.

Great! 🥳  Thanks for confirming @japsmits! 🎖️